SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1555,7 +1554,10 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 
 		if (LoadBalanceEnforcer.IsValid())
 		{
-			LoadBalanceEnforcer->Tick();
+			for(const auto& elem : LoadBalanceEnforcer->ProcessQueuedAclAssignmentRequests())
```

Comment:
Not sure if ProcessQueuedAclAssignmentRequests is now the correct function name for this? Maybe it is, it just made me think.

Reply:
I agree, there might be a better name. It still does some processing internally though, so I kept it as is for now. What name could you suggest?